airflow-core/docs/tutorial/taskflow.rst

```diff
@@ -19,7 +19,7 @@
 Pythonic DAGs with the TaskFlow API
 ===================================
 
-In the first tutorial, you built your first Airflow DAG using traditional Operators like ``PythonOperator``.
+In the first tutorial, you built your first Airflow DAG using traditional Operators like ``BashOperator``.
```

Comment:
Why was this changed? What was wrong? @AzisK ?

Reply:
The first tutorial has no usage of `PythonOperator` https://airflow.apache.org/docs/apache-airflow/stable/tutorial/fundamentals.html

We can solve this in 2 ways

1) Currently offered solution saying that we built the first tutorial using `BashOperator`s
2) Build a second tutorial which would use `PythonOperator`s and then "Pythonic DAGs with the TaskFlow API" would be the third tutorial

2nd way is more work but I could work on it if you agree? @kaxil

Reply:
aah makes sense, all good 👍